processors/blockprocessor/sample.yaml

```diff
@@ -0,0 +1,2 @@
+Name: block_evaluator
+Config:
```

Comment:
I like this approach, it gives us a lot of flexibility to include inline documentation:

```suggestion
# Execute block transactions to compute the new account states.
name: block_evaluator
config:
    # each block evaluator field and its comment.
```

We should add a unit test for each of these and make sure:
* every field is set (i.e. make sure the sample is updated when a new field is added.
* the sample is definitely valid.

Maybe there's a way to use reflection to make a generic function that accepts the string and the concrete type?

Reply:
I can make sure that the sample is valid by initializing a plugin and using `init()` to consume and check for errors.  I'm not sure how to do the other part where we check that every field is filled out.  I'll look at generics/reflection to see what we can do.